internal/pkg/cmd/prerunner_test.go

```diff
@@ -184,15 +184,13 @@ func TestPreRun_TokenExpires(t *testing.T) {
 	r := getPreRunBase()
 	r.Config = cfg
 
-	root := &cobra.Command{
-		Run: func(cmd *cobra.Command, args []string) {},
-	}
-	rootCmd := pcmd.NewAnonymousCLICommand(root, r)
+	root := &cobra.Command{Run: func(cmd *cobra.Command, args []string) {}}
+	rootCmd := pcmd.NewAuthenticatedCLICommand(root, r)
 	root.Flags().CountP("verbose", "v", "Increase verbosity")
 	root.Flags().Bool("unsafe-trace", false, "")
 
 	_, err := pcmd.ExecuteCommand(rootCmd.Command)
-	require.NoError(t, err)
+	require.Error(t, err)
```

Comment:
😨

Reply:
This is intended! We have to force an error to happen (corresponds to the backend returning `""` for the auth token) because if the auth token exists, it'll try to send it to the backend which we don't have mocks for in our unit tests.